src/openapi/data-ingestion-schema-v1.yaml

```
@@ -1948,7 +1960,8 @@ components:
         id:
           type: string
           description: |
-            Product option ID. For `CONFIGURABLE` or `SWATCH` option types, this ID must be match the ["attribute code"](#operation/ProductsPost!path=attributes/code&t=request) used for the configurable product (for example, "color").
+            Product option ID. For `CONFIGURABLE` or `SWATCH` option types, this ID must be match the
+            ["attribute code"](#operation/ProductsPost!path=attributes/code&t=request) used for the configurable product (for example, "color").
```

```suggestion
            Product option ID. For `CONFIGURABLE` or `SWATCH` option types, this ID must match the
            ["attribute code"](#operation/ProductsPost!path=attributes/code&t=request) used for the configurable product (for example, "color").
```

This might be worth fixing in the schema.